packages/kernel/src/declarations.d.ts

```diff
@@ -19,18 +19,6 @@ declare module "!!raw-loader!*" {
   export default res;
```

Comment:
`raw-loader` can also be deleted.

Reply:
This was introduced when we imported the worker code with raw-loader: https://github.com/whitphx/stlite/commit/c2e3626b2999b09c0e8f42ae615b641886d1eb88#diff-845c357d4b52d91a12c4d528168b3a89ce8950a407322aef7ef592bc9c1f570cR5

It's no longer used.